legate/core/_legion/operation.py

```diff
@@ -1145,14 +1145,16 @@ def __init__(
         self._launcher = ffi.gc(
             self.launcher, legion.legion_attach_launcher_destroy
         )
+        if not data.c_contiguous and not data.f_contiguous:
+            raise RuntimeError("Can only attach to C- or F-contiguous buffers")
```

Comment:
Woudn't `not data.contiguous` also suffice here? Python docs for [`contiguous`](https://docs.python.org/3/glossary.html#term-contiguous) state

> A buffer is considered contiguous exactly if it is either C-contiguous or Fortran contiguous.

Reply:
You're right, I hadn't looked into this very carefully. There are (generally speaking) ways to lay out a 3+ dimension array such that it's "contiguous", but neither in C- nor Fortran-order (e.g. by ordering the dimensions `x > z > y`), but it looks like cpython doesn't consider this case.

Reply:
Changed to just checking `.contiguous` with d891466071d52d98979134b7e171ed7e5d2b33ae